community/rust/APKBUILD

```diff
@@ -3,10 +3,10 @@
 # Contributor: Jeizsm <jeizsm@gmail.com>
 # Maintainer: Jakub Jirutka <jakub@jirutka.cz>
 pkgname=rust
-pkgver=1.31.1
+pkgver=1.32.0
 # TODO: bump to 6 as soon as we add llvm6
```

Comment:
LLVM 7 has been added in #5717

Reply:
Thx, i tried to compile 1.32 with llvm7 and lto test failed, after that I found these issue. https://github.com/rust-lang/rust/issues/57762
So it's needed to add these patches to alpine llvm7 to use it with rust
https://github.com/llvm-mirror/llvm/commit/da1fb72
https://github.com/llvm-mirror/llvm/commit/cc1f2a5

Reply:
Would this be fixed by LLVM 8?

Reply:
Yes, but patches already in alpine llvm7 https://github.com/alpinelinux/aports/pull/6892
I'm trying to build rust with it and if everything ok, I will update this pull request with llvm7